fix dnssec test

danawillow opened the pull request:
<!-- 
Note: You may see "This branch is out-of-date with the base branch"
when you submit a pull request. This is fine! We don't use the GitHub
merge button to merge PRs, and you can safely ignore that message.
-->

<!-- Your regular pull request description goes here -->
Moved dnssec into its own test since it can't be used with private zones and also can't be updated (actually I'm pretty sure that it can, but that should be fixed in a future PR. My impression from looking at the docs is that all fields can be updated, but that needs testing and I don't feel like doing that right now).



<!--
For each repository you expect to modify with this PR, fill in a repo-specific
PR title under the corresponding tag. We use repo-specified PR titles to ensure
that each downstream has a clear, easy to understand history.

If the Magician generates a PR for a repo with no specified title, it will use
the title of this PR. [terraform-beta] will inherit the title of [terraform]
if it has no specified title.
-->

<!-- Optional PR titles that describe your downstream changes -->
-----------------------------------------------------------------
# [all]
## [terraform]
### [terraform-beta]
## [ansible]
## [inspec]

modular-magician commented:
Hi!  I'm the modular magician, I work on Magic Modules.
This PR seems not to have generated downstream PRs before, as of f247ef13283253932e0afd72600c8ed63315a084. 
## Pull request statuses
No diff detected in terraform-google-conversion.
No diff detected in Inspec.
## New Pull Requests
I built this PR into one or more new PRs on other repositories, and when those are closed, this PR will also be merged and closed.
depends: https://github.com/terraform-providers/terraform-provider-google-beta/pull/777 
depends: https://github.com/terraform-providers/terraform-provider-google/pull/3735 
depends: https://github.com/modular-magician/ansible/pull/267

tysen commented on products/dns/api.yaml:
```diff
@@ -81,7 +81,8 @@ objects:
             name: 'defaultKeySpecs'
             description: |
               Specifies parameters that will be used for generating initial DnsKeys
-              for this ManagedZone. Output only while state is not OFF
```
Is it still important to note that it's "Output only while state is not OFF"?

danawillow replied:
Added a comment to add a note back into the descriptions, since it isn't relevant until we have update support.

modular-magician commented:
Hi!  I'm the modular magician, I work on Magic Modules.
I see that this PR has already had some downstream PRs generated. Any open downstreams are already updated to your most recent commit, c40458b1d7dec0655323aee69838484e50091423. 
## Pull request statuses
terraform-provider-google-beta already has an open PR.
No diff detected in terraform-google-conversion.
terraform-provider-google already has an open PR.
Ansible already has an open PR.
No diff detected in Inspec.
## New Pull Requests
I didn't open any new pull requests because of this PR.

danawillow commented:
Your comment made me look further into what that statement meant, and when which fields are returned from the API. Marked another field as computed, added a test, and changed the wording around a little- mind taking a quick second pass?
